guide/14-deep-learning/yolov3_object_detector.ipynb

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
<p>These models are already a part of arcgis python API change to ArcGIS API for Python.<strong> </strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/881/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>